.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,7 +16,8 @@
 * **What is the new behavior?**
 <!-- if this is a feature change -->
 
-- **Does this PR introduce a breaking change?**
+- **Does this PR introduce a breaking change, and is
+[labeled accordingly](https://arduino.github.io/arduino-cli/latest/CONTRIBUTING/#pull-requests)?**
```

Comment:
I wonder it the term "labeled" is clear enough? It might lead people to think of GitHub's labeling feature.

I have two ideas of how things could be made more clear:
- Change the term to something like "titled".
- Add an anchor to the specific bullet point in the contributing guidelines, and link to that:
    ```md
    - <a id="breaking"></a>Please start the commit message and PR title with the string **[breaking]** if the PR contains a breaking change. A
    ```